query/src/sql/sql_parser.rs

```diff
@@ -948,8 +948,7 @@ impl<'a> DfParser<'a> {
                 .parse_keywords(&[Keyword::IF, Keyword::NOT, Keyword::EXISTS]);
 
         let udf_name = self.parser.parse_literal_string()?;
-        let as_token = Token::make_keyword("AS");
-        self.parser.expect_token(&as_token)?;
+        self.parser.expect_keyword(Keyword::AS)?;
```

Comment:
It would be better to add a test like:
https://github.com/datafuselabs/databend/blob/bf989b9bae6234518937814e57692ae5a9313205/query/tests/it/sql/sql_parser.rs#L1409-L1418

```
 expect_parse_ok( 
     "CREATE FUNCTION test_udf as (p, d) -> not(isnotnull(p, d)) DESC = 'this is a description'", 
     DfStatement::CreateUDF(DfCreateUDF { 
         if_not_exists: false, 
         udf_name: "test_udf".to_string(), 
         parameters: vec!["p".to_string(), "d".to_string()], 
         definition: "not(isnotnull(p,d))".to_string(), 
         description: "this is a description".to_string(), 
     }), 
 )?; 
```

Reply:
done